### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Added a Context block to SlackMessage class and used it in Delivery Mechanism.
Looks a bit better, might as well use it

image

### CODE REVIEW

File: CryptoTechProject/DeliveryMechanism.cs [hunk 1]
```diff
@@ -12,8 +12,8 @@ public class DeliveryMechanism
 
         public void Run(Action onStarted)
         {
-            httpListener.Prefixes.Add($"http://+:{System.Environment.GetEnvironmentVariable("PORT")}/");
-            //httpListener.Prefixes.Add("http://localhost:5000/");
+            //httpListener.Prefixes.Add($"http://+:{System.Environment.GetEnvironmentVariable("PORT")}/");
```
Comment (craigjbass) [inline CryptoTechProject/DeliveryMechanism.cs]: Commented code?

File: CryptoTechProject/DeliveryMechanism.cs [hunk 2]
```diff
@@ -22,11 +22,11 @@ public void Run(Action onStarted)
...
 
                 //HardCodedWorkshopsGateway gateway = new HardCodedWorkshopsGateway();
-                AirtableGateway gateway = new AirtableGateway(System.Environment.GetEnvironmentVariable("AIRTABLE_URL"),
+               /*AirtableGateway gateway = new AirtableGateway(System.Environment.GetEnvironmentVariable("AIRTABLE_URL"),
                     System.Environment.GetEnvironmentVariable("AIRTABLE_API_KEY"),
-                    System.Environment.GetEnvironmentVariable("AIRTABLE_TABLE_ID"));
+                    System.Environment.GetEnvironmentVariable("AIRTABLE_TABLE_ID"));*/
                 
-                //AirtableGateway gateway = new AirtableGateway("https://api.airtable.com/", INSERT_API_KEY, INSERT_TABLE_ID);
+                AirtableGateway gateway = new AirtableGateway("https://api.airtable.com/", "keyZqU8PXE97s45Jw", "apppGaPfIFW8nmf4T");
```
Comment (craigjbass) [inline CryptoTechProject/DeliveryMechanism.cs]: Secret key!

Comment (danielburnley) [inline CryptoTechProject/DeliveryMechanism.cs]: :eyes:

Comment (ccp92) [inline CryptoTechProject/DeliveryMechanism.cs]: 🚨is this your airtable api key?
You should take a look into using environment variables to hide these. Especially on a public repo!

Comment (kosiakkatrina) [reply CryptoTechProject/DeliveryMechanism.cs]: Why are yalls so quick, I'm rotating it no sweat :'(

Comment (kosiakkatrina) [reply CryptoTechProject/DeliveryMechanism.cs]: Yeah, I'm so sorry :< Keep on forgetting it cause was testing locally, it's rotated now

**Pull request merged**